js/index.js

```diff
@@ -14,29 +14,112 @@ const siteContent = {
-    "copyright" : "Copyright Great Idea! 2018"
+    "copyright": "Copyright Great Idea! 2018"
   },
 };
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute('src', siteContent["nav"]["img-src"]);
+
+// Nav links
+navItemsArray = Object.values(siteContent.nav);
+
+const navlinks = document.querySelectorAll('nav a');
+
+for (let i = 0; i < navItemsArray.length - 1; i++) {
+  navlinks[i].textContent = navItemsArray[i];
+  
+  // Change color to nav links to green
+  navlinks[i].style.color = 'green';
+}
+
+// Prepend and append child elements to the nav links
+const append = document.createElement('a');
+const prepend = document.createElement('a');
+const nav = document.querySelector('nav');
+
+nav.appendChild(append);
+nav.prepend(prepend);
+
+append.textContent = 'Append';
+prepend.textContent = 'Prepend';
+append.style.color = 'green';
+prepend.style.color = 'green';
+
+
+
+// CTA section
+const ctaH1 = document.querySelector('.cta-text h1');
+
+ctaH1.innerHTML = `${siteContent.cta.h1.slice(0, 3)}<br>${siteContent.cta.h1.slice(4, 6)}<br>${siteContent.cta.h1.slice(7, 14)}`;
+
+const ctaBtn = document.querySelector('.cta-text button');
+
+ctaBtn.textContent = siteContent.cta.button;
+
+const ctaImg = document.querySelector('#cta-img');
+
+ctaImg.src = siteContent.cta['img-src'];
+
+
+// Main Content Section
+
+const mainContentH4 = document.querySelectorAll('.text-content h4');
+const mainContentP = document.querySelectorAll('.text-content p');
+const middleImg = document.querySelector('.main-content img');
+
+mainContentH4[0].textContent = siteContent["main-content"]['features-h4'];
+mainContentH4[1].textContent = siteContent["main-content"]['about-h4'];
+mainContentH4[2].textContent = siteContent["main-content"]['services-h4'];
+mainContentH4[3].textContent = siteContent["main-content"]['product-h4'];
+mainContentH4[4].textContent = siteContent["main-content"]['vision-h4'];
+
+mainContentP[0].textContent = siteContent['main-content']['features-content'];
+mainContentP[1].textContent = siteContent['main-content']['about-content'];
+mainContentP[2].textContent = siteContent['main-content']['services-content'];
+mainContentP[3].textContent = siteContent['main-content']['product-content'];
+mainContentP[4].textContent = siteContent['main-content']['vision-content'];
+
+middleImg.src = siteContent['main-content']['middle-img-src'];
+
+
+// Contact Section
+
+const contactH4 = document.querySelector('.contact h4');
+const contactP = document.querySelectorAll('.contact p');
+
+contactH4.textContent = siteContent.contact['contact-h4'];
+contactP[0].textContent = siteContent.contact.address;
+contactP[1].textContent = siteContent.contact.phone;
+contactP[2].textContent = siteContent.contact.email;
+
+
+// Footer Section
+const footerP = document.querySelector('footer p');
+
+footerP.textContent = siteContent.footer.copyright;
+
+
+
+
+
```

Comment:
`// Example: Update the img src for the logo
let logo = document.getElementById("logo-img");
logo.setAttribute('src', siteContent["nav"]["img-src"])

// REMEMBER, you can accomplish the assignment in many different ways. You could literally use getElementByTag name to accomplish everything if you wanted:

// Update the nav links test
let selectedNavLinks = document.querySelectorAll("nav a");
selectedNavLinks.forEach((link, i) => {
  link.innerHTML = siteContent.nav[`nav-item-${i+1}`];
})

// Update CTA
let ctaText = document.getElementsByClassName("cta-text")[0];

ctaText.getElementsByTagName("h1")[0].innerHTML = siteContent["cta"]["h1"];
ctaText.getElementsByTagName("button")[0].innerHTML = siteContent["cta"]["button"];

let ctaImg = document.getElementById("cta-img");
ctaImg.setAttribute("src", siteContent["cta"]["img-src"]);
// console.log(document.getElementsByClassName("cta")[0])

// Update MainContent
let textContentList = document.querySelectorAll(".text-content");

textContentList[0].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["features-h4"];
textContentList[0].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["features-content"];
textContentList[1].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["about-h4"];
textContentList[1].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["about-content"];
textContentList[2].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["services-h4"];
textContentList[2].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["services-content"];
textContentList[3].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["product-h4"];
textContentList[3].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["product-content"];
textContentList[4].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"]["vision-h4"];
textContentList[4].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]["vision-content"];

let middleImg = document.getElementById("middle-img");
middleImg.setAttribute('src', siteContent["main-content"]["middle-img-src"])

// Update Contact
let contact = document.getElementsByClassName("contact")[0];
contact.getElementsByTagName("h4")[0].innerHTML = siteContent["contact"]["contact-h4"]
contact.getElementsByTagName("p")[0].innerHTML = siteContent["contact"]["address"]
contact.getElementsByTagName("p")[1].innerHTML = siteContent["contact"]["phone"]
contact.getElementsByTagName("p")[2].innerHTML = siteContent["contact"]["email"]

// Update Footer
let footer = document.querySelector("footer");
footer.getElementsByTagName("p")[0].innerHTML = siteContent["footer"]["copyright"];

// Add New Content
// Change navigation text color
selectedNavLinks.forEach((link,i) => {
  link.style.color = "green";
})

// Add two items to navigation
selectedNavLinks = document.getElementsByTagName("nav")[0];
let createNewNode = (name) => {
  let newNode = document.createElement("a");
  newNode.innerHTML = name;
  return newNode;
}
selectedNavLinks.prepend(createNewNode("Extra Item 1"));
selectedNavLinks.append(createNewNode("Extra Item 2"));`